src/python/espressomd/system.pyx

```diff
@@ -359,6 +360,63 @@ cdef class System:
 
         return make_array_locked(mi_vec)
 
+    def dist_to(self, id=None, pos=None):
```

Comment:
the name of this function is very inexpressive. Should this function actually exist? If you have a function that gives you the distance between two positions, say ```dist(pos1, pos2)```, you can just to ```min_dist = np.min([dist(pos1, p.pos) for p in system.part])```. So if it's a not often used one-liner in python, we probably don't need a function for it?

Reply:
in my notes of the meeting, I wrote "dist_to? should be with system.distance", but maybe I didn't catch what you said about removing it

Reply:
yeah sorry, it just got clear to me looking at the code

Reply:
would  you like me to change the `system.distance_vec()` to take either particles or 3d vectors, so that we can use your one-liner?